service/frontend/configs/quotas.go

```diff
@@ -140,6 +140,11 @@ var (
 		"ListBatchOperations":    1,
 		"StartBatchOperation":    1,
 		"StopBatchOperation":     1,
+
+		// TODO: added temporarily here; need to be check when implemented if it's the right place and priority
```

Comment:
@bergundy is this correct?

Reply:
Don't worry about this, these APIs are not implemented.